plugins/chat/app/services/service/base.rb

```diff
@@ -127,6 +127,10 @@ class ModelStep < Step
       def call(instance, context)
         context[name] = super
         raise ArgumentError, "Model not found" if context[name].blank?
+        if context[name].try(:invalid?)
```

Comment:
Using `#try` here so this step is still compatible with non AR models (if `#invalid?` isn’t available, it won’t do anything contrarily to the safe navigation operator).